pkg/apis/crd/v1alpha2/types.go

```diff
@@ -306,28 +306,40 @@ type IPPoolSpec struct {
 	IPRanges []SubnetIPRange `json:"ipRanges"`
 }
 
-// SubnetIPRange is a set of contiguous IP addresses, represented by a CIDR or a pair of start and end IPs,
-// along with subnet definition.
-type SubnetIPRange struct {
-	IPRange `json:",inline"`
+// SubnetSpec is subnet specification for IP Range
+type SubnetSpec struct {
```

Comment:
I feel we should not call it SubnetSpec as it does not really define a subnet, but just some attributes of a subnet. No good idea myself though.

Reply:
Perhaps SubnetInfo?

Reply:
Sounds better than SubnetSpec.

Reply:
Done